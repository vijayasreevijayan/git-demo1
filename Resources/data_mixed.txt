1,13,19.876
9.45, 12, 8.3, 7
3, 5, 6, 7, 9.8