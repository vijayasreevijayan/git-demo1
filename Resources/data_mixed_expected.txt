data_mixed.txt
Line 1 average 11.292 standard deviation 9.55321
Line 2 average 9.1875 standard deviation 2.12539
Line 3 average 6.16 standard deviation 2.51555